scripts/build-test-lint.sh

```diff
@@ -5,7 +5,7 @@
 set -eu
 
 echo "Checking that it builds"
-gb build
+go build
```

Comment:
This will check that everything builds right? `./build.sh` is just for outputting all the different binaries?

Reply:
Yep. This line is useless though, as I just realised the right thing is done below it.